src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -0,0 +1,237 @@
+        #
+        # @return [Boolean]
+        def selected_last_fstab?
+          selected_fstab == fstabs.last
+        end
+
+        # Errors in the selected fstab
+        #
+        # @see #not_importable_entries_error
+        #
+        # @return [Array<String>]
+        def selected_fstab_errors
+          [not_importable_entries_error].compact
+        end
+
+        # Imports mount points from the selected fstab
+        #
+        # Before importing, the current devicegraph is reset to the system one.
+        def import_mount_points
+          reset_current_graph
+          importable_entries.each { |e| import_mount_point(e) }
+        end
+
+      private
+
+        # System devicegraph
+        #
+        # @return [Y2Storage::Devicegraph]
+        def system_graph
+          DeviceGraphs.instance.system
+        end
+
+        # Current devicegraph
+        #
+        # @return [Y2Storage::Devicegraph]
+        def current_graph
+          DeviceGraphs.instance.current
+        end
+
+        # Disk analyzer for the system devicegraph
+        #
+        # @return [Y2Storage::DiskAnalyzer]
+        def disk_analyzer
+          DeviceGraphs.instance.disk_analyzer
+        end
+
+        # Error when some entries in the selected fstab cannot be imported
+        #
+        # An entry cannot be imported when the device is not found or it is used
+        # by other device (e.g., used by LVM or MD RAID).
+        #
+        # @return [String, nil] nil if all entries can be imported
+        def not_importable_entries_error
+          entries = not_importable_entries
+          return nil if entries.empty?
+
+          mount_points = entries.map(&:mount_point).join("\n")
+
+          format(_("The following mount points cannot be imported:\n%{mount_points}"),
+            mount_points: mount_points)
+        end
+
+        # Entries in the current selected fstab that can be imported
+        #
+        # @see #can_be_imported?
+        #
+        # @return[Array<Y2Storage::SimpleEtcFstabEntry>]
+        def importable_entries
+          selected_fstab.filesystem_entries.select { |e| can_be_imported?(e) }
+        end
+
+        # Entries in the current selected fstab that cannot be imported
+        #
+        # @see #can_be_imported?
+        #
+        # @return[Array<Y2Storage::SimpleEtcFstabEntry>]
+        def not_importable_entries
+          selected_fstab.filesystem_entries - importable_entries
+        end
+
+        # Whether a fstab entry can be imported
+        #
+        # An entry can be imported when the device is known and it is not used
+        # by other device (e.g., used by LVM or MD RAID) or it is a known NFS.
+        #
+        # @param entry [Y2Storage::SimpleEtcFstabEntry]
+        # @return [Boolean]
+        def can_be_imported?(entry)
+          device = entry.device(system_graph)
+          return false unless device
+
+          !device.is?(:blk_device) || can_be_formatted?(device)
```

Comment:
Why the `!device.is?(:blk_device)` check? I would appreciate an explanatory comment

Reply:
I tried to explain it the description of the method: An entry can be imported when the device is known and it is not used by other device (e.g., used by LVM or MD RAID), or it is a known NFS.", but I put it more explicitly in the code line.